docs/superpowers/plans/2026-05-03-bundled-bug-fixes-30-31-32-33.md

```
@@ -0,0 +1,826 @@
+
+```bash
+pidof nwg-notifications && echo "(daemon up with new binary)"
+```
+
+- [ ] **Step 2: Hand off to the user — STOP HERE**
+
+Tell the user (verbatim or close):
+> Installed and restarted via `make upgrade`. Smoke-test paths for each bug:
+>
+> **#32 — GetServerInformation:**
+> ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)